README.md

```diff
@@ -6,18 +6,17 @@ links that you post or send to your team members, collects and organizes them fo
 **To run Linkbot:**
 
 * Clone the repo && `bundle install` && `rake db:create db:migrate`
```

Comment:
Having missed the second and third parts of the first step, might it be better to write it out with numerical steps instead of the one-liner here?